src/scope.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @ignore
+ */
+const dedupe = arr => arr.filter((x, i) => arr.indexOf(x) === i);
+
+/**
+ * @ignore
+ */
+export const getUniqueScopes = (...scopes: string[]) => {
+  const scopeString = scopes.filter(Boolean).join();
```

Comment:
(optional) since `split` takes a regex you could simplify to:
`return scopes.join(' ').split(/\s+/).join(' ')`

Reply:
Also needed a `trim()` in there, but yes - implemented this.